lib/flame_audio.dart

```diff
@@ -0,0 +1,18 @@
+import 'package:audioplayers/audioplayers.dart';
+import 'package:audioplayers/audio_cache.dart';
+
+class FlameAudio {
+  AudioCache audioCache = AudioCache(prefix: 'audio/');
+
+  Future<AudioPlayer> play(String file) {
+    return audioCache.play(file);
+  }
+
+  Future<AudioPlayer> loop(String file) {
```

Comment:
for example, the loop is not clear to me which api it will use without checking the code. what about loopMusic?